src/envs/cover.py

```diff
@@ -366,20 +366,22 @@ class CoverMultistepOptions(CoverEnvTypedOptions):
     over a target region. We should fix this to allow dropping
     blocks anywhere. Also, would be good to check collisions.
```

Comment:
please update this comment too, thanks! (also just to check my understanding -- the first two sentences here are still true right? just the third sentence is now obsolete. that's all good, just making sure.)